pandas/tests/frame/test_reductions.py

```diff
@@ -1025,6 +1025,27 @@ def test_idxmax_mixed_dtype(self):
         expected = Series([0, 2, 1, 2], index=[1, 2, 3, 4])
         tm.assert_series_equal(result, expected)
 
+        # with convert dtypes
```

Comment:
Please make a new test for this.

Could you try a parametrization?

Reply:
sure thing, I'll separate them out and use parametrization